tf2_ros/include/tf2_ros/static_transform_broadcaster.h

```diff
@@ -52,6 +52,8 @@ class StaticTransformBroadcaster{
   /** \brief Constructor (needs a ros::Node reference) */
   StaticTransformBroadcaster();
```

Comment:
I think that this constructor prototype should be removed (or else implemented with a internal creation of a node, but I think that it's clearer to force the user to pass in a node as is done with the other constructor).